source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1054,6 +1054,15 @@ export class CardElementPeer extends DesignerPeer {
             }
         }
 
+        if (getExcludedProperties.indexOf("isVisible") < 0) {
+            let isVisible = addLabelAndInput(card, "Initially visible:", Adaptive.ToggleInput);
+            isVisible.input.defaultValue = String(this.cardElement.isVisible);
+            isVisible.input.onValueChanged = () => {
+                this.cardElement.isVisible = isVisible.input.value == "true";
```

Comment:
this is toggleVisibility though not quite related to richtextbox yes?

Reply:
Opportunistic bug fixing